teamA/lib/Views/send_quiz_to_moodle.dart

```diff
@@ -0,0 +1,447 @@
+  List<String> hours =
+      List.generate(24, (index) => index.toString().padLeft(2, '0'));
+  List<String> minutes =
+      List.generate(60, (index) => index.toString().padLeft(2, '0'));
+
+  // Initial selected value for category
+  String dropdownvalue = 'No selection';
+
+  // List of categories in the dropdown menu
+  var items = [
+    'No selection',
+    'Category 1',
+    'Category 2',
+    'Category 3',
+    'Category 4',
+  ];
+
+  // List of attempts options
+
+  var items2 = [
+    'No selection',
+    'Unlimited',
+    'First',
+    'Second',
+    'Last',
+  ];
+
+  //List of grading methods
+  var items3 = [
+    'No selection',
+    'Highest Grade',
+    'Average Grade',
+    'Low Grade',
+  ];
+
+  TextEditingController courseNameController = TextEditingController();
+  TextEditingController quizNameController = TextEditingController();
+  TextEditingController gradeController = TextEditingController();
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+        home: new Scaffold(
+      appBar: AppBar(
+        centerTitle: true,
+        backgroundColor: Theme.of(context)
+            .colorScheme
+            .primaryContainer, // Use primary container color
+        elevation: 0,
+        flexibleSpace: SafeArea(
+            child: Row(
+          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+          children: [
+            IconButton(
+              icon: Icon(Icons.arrow_back),
+              onPressed: () {
+                Navigator.push(
+                  context,
+                  MaterialPageRoute(builder: (context) => DevLaunch()),
+                );
+                //to do something once pressed
+              },
+            ),
+            Expanded(
+              child: Align(
+                alignment: Alignment.center,
+                child: Text(
+                  'Learning Lens',
+                  style: TextStyle(
+                    color: Theme.of(context).colorScheme.onPrimaryContainer,
+                    fontSize: 20,
+                    fontWeight: FontWeight.bold,
+                  ),
+                ),
+              ),
+            ),
+          ],
+        )),
+      ), // Use primary container color
+      body: SingleChildScrollView(
+        padding: EdgeInsets.all(15.0),
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            Center(
+              child: Padding(
+                padding: const EdgeInsets.only(top: 15.0),
+                child: Text(
+                  'Send Quiz to Moodle',
+                  textDirection: TextDirection.ltr,
+                  style: TextStyle(fontSize: 25, fontWeight: FontWeight.normal),
+                  textAlign: TextAlign.center,
+                ),
+              ),
+            ),
+            SizedBox(
+              height: 30,
+            ),
+
+            sectionTitle(title: 'Quiz Information'),
+            TextField(
```

Comment:
Course name should really be a dropdown with its contents taken from the course list, but that can be handled as part of the BL.